### PR TITLE
Add kubernetes config files

### PR DESCRIPTION
**Pull request by trisberg**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,6 +18,10 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-actuator</artifactId>
```
Comment (fbiville) [inline pom.xml]: 👍 for this
is it a nice to have or is it required for K8s (for K8s probes, I'd assume?)

Comment (trisberg) [reply pom.xml]: need it for the probes

File: README.md [hunk 2]
```diff
@@ -34,6 +34,37 @@ Run the inventory app with a `cloud` profile:
 java -jar target/inventory-api-0.0.1-SNAPSHOT.jar --spring.profiles.active=cloud
 ```
 
+### Use a PostgreSQL database on Kubernetes
+
+Build the Inventory API and push to DockerHub.
+
+```shell script
+pack build -p . projectriff/inventory-api
```
Comment (fbiville) [inline README.md]: I'm using pack v0.4.1 and it's telling to select a builder before it can do anything.
What version are you running?

Comment (trisberg) [reply README.md]: I have `v0.5.0`, but yeas I have a default builder configured

Comment (trisberg) [reply README.md]: I'm using `cloudfoundry/cnb:cflinuxfs3`

**Comment by fbiville** [comment]
Merged as 607abb8e3f237f3de3aef2a7cd8926b0882c5d72

**Pull request closed without merging**